uber/tests/conftest.py

```diff
@@ -3,7 +3,11 @@
 import pytest
 from sideboard.tests import patch_session
 
-TEST_DB_FILE = '/tmp/uber.db'
+
+try:
+    TEST_DB_FILE = c.TEST_DB_FILE
+except AttributeError:
+    TEST_DB_FILE = '/tmp/uber.db'
```

Comment:
This can probably be a lot better, but `c.TEST_DB_FILE` will only exists if `test-defaults.ini` was loaded.